automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+import time
+
+from tools import reboot
+from automation_tools import setup_capsule_firewall
+from automation_tools.repository import enable_repos, disable_repos
+from automation_tools.satellite6.capsule import generate_capsule_certs
+from automation_tools.utils import distro_info, update_packages
+from fabric.api import env, execute, run
+from tasks import (
+    create_rhevm_instance,
+    delete_rhevm_instance,
+    sync_capsule_repos_to_upgrade
+)
+from tools import copy_ssh_key
+
+
+def satellite6_capsule_setup(sat_host, os_version):
+    """Sets up required things on upgrade running machine and on Capsule to
+    perform capsule upgrade later
```

Comment:
`Setup all per-requisites for user provided capsule or auto created capsule on rhevm for capsule upgrade.` I think this gives us more clarity as what this module does